gdnative-derive/src/methods.rs

```diff
@@ -196,122 +221,138 @@ fn impl_gdnative_expose(ast: ItemImpl) -> (ItemImpl, ClassMethodExport) {
-                            let _export_args = export_args.get_or_insert_with(ExportArgs::default);
-                            if !attr.tokens.is_empty() {
-                                use syn::{Meta, MetaNameValue, NestedMeta};
-
-                                let meta = match attr.parse_meta() {
-                                    Ok(val) => val,
-                                    Err(err) => {
-                                        errors.push(err);
-                                        return false;
-                                    }
-                                };
+                        let (is_export, is_old_syntax) = if let Some("export") = last_seg.as_deref()
+                        {
+                            (true, true)
+                        } else if let Some("method") = last_seg.as_deref() {
+                            (true, false)
+                        } else {
+                            (false, false)
+                        };
 
-                                let pairs: Vec<_> = match meta {
-                                    Meta::List(list) => list
-                                        .nested
-                                        .into_pairs()
-                                        .filter_map(|p| {
-                                            let span = p.span();
-                                            match p.into_value() {
-                                                NestedMeta::Meta(Meta::NameValue(pair)) => {
-                                                    Some(pair)
-                                                }
-                                                unexpected => {
-                                                    let msg = format!(
-                                                        "unexpected argument in list: {}",
-                                                        unexpected.into_token_stream()
-                                                    );
-                                                    errors.push(syn::Error::new(span, msg));
-                                                    None
-                                                }
-                                            }
-                                        })
-                                        .collect(),
-                                    Meta::NameValue(pair) => vec![pair],
-                                    meta => {
-                                        let span = meta.span();
-                                        let msg = format!(
-                                            "unexpected attribute argument: {}",
-                                            meta.into_token_stream()
-                                        );
+                        if is_export {
+                            let mut _export_args =
+                                export_args.get_or_insert_with(ExportArgs::default);
```

Comment:
Don't start variables with `_` if you refer to them. It's perfectly fine to reassign the same name, i.e. 
```rs
let mut export_args =
    export_args.get_or_insert_with(ExportArgs::default);
export_args.is_old_syntax = is_old_syntax;
```